src/app/page.tsx

```diff
@@ -1,8 +1,38 @@
+"use client"
```

Comment:
how we can make page.tsx as client component , can you read and check about this

Reply:
We can make all the components in which useState and useEffect are used as client components.

Reply:
The intention is to make use of next js features with focus on SSR and not just follow react way of coding.